src/marqo/tensor_search/tensor_search.py

```diff
@@ -1702,8 +1800,8 @@ def _vector_text_search(
     else:  # is dict:
         ordered_queries = list(query.items())
         if index_info.index_settings[NsField.index_defaults][NsField.treat_urls_and_pointers_as_images]:
-            text_queries = [k for k, _ in ordered_queries if _is_image(k)]
```

Comment:
It seems this has always been a bug. What affect does this have, just a naming switch?

Reply:
I think this had no real effect on the execution, as the point of this part of the code was just to separate image and text queries from each other. Just a naming switch, as you said. But this caused an issue when it was refactored into bulk search (there were issues when checking for the jobs' content-type)